.github/workflows/coding-standards.yml

```diff
@@ -52,9 +52,10 @@ jobs:
         uses: "actions/cache@v3"
         with:
           path: ".php_cs.cache"
-          key: "composer-${{ runner.os }}-${{ matrix.php-version }}-${{ hashFiles('composer.*') }}"
+          key: "composer-${{ runner.os }}-${{ matrix.php-version }}-${{ github.ref_name }}"
```

Comment:
For reference, see https://docs.github.com/en/actions/learn-github-actions/contexts.